fixtures/sumger/dm-rgb400.json

```diff
@@ -0,0 +1,472 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "DM-RGB400",
+  "shortName": "DM-RGB400",
+  "categories": ["Laser"],
+  "meta": {
+    "authors": ["Jason dinAlt"],
+    "createDate": "2020-07-02",
+    "lastModifyDate": "2020-07-02"
+  },
+  "comment": "The documentation is pretty sketchy on this device. A fair amount of the information was derived experimentally.",
+  "links": {
+    "manual": [
+      "http://www.bsl-lighting.com/wp-content/uploads/2015/01/FLASH-RGB-400-Manual.pdf"
```

Comment:
I'm not sure there's much value in keeping this URL.  The name/model doesn't quite match the others used here, and the DMX mapping is very different.  Is this really the same fixture?